src/rules/unnecessaryElseRule.ts

```diff
@@ -27,9 +27,16 @@ export class Rule extends Lint.Rules.AbstractRule {
         description: Lint.Utils.dedent`
         Disallows \`else\` blocks following \`if\` blocks ending with a \`break\`, \`continue\`, \`return\`, or \`throw\` statement.`,
         descriptionDetails: "",
-        optionExamples: [true],
-        options: null,
-        optionsDescription: "Not configurable.",
+        optionExamples: [true, [true, { allowElseIf: true }]],
```

Comment:
> allowElseIf

TSLint rules use dash-case, so this should be `allow-else-if`. For example,  `noAnyRule.ts` has `const OPTION_IGNORE_REST_ARGS = "ignore-rest-args";`.

Reply:
Done https://github.com/palantir/tslint/pull/4599/commits/03b2414e52f35c0e035b78ede3496dc04839550e